pypeit/core/coadd.py

```diff
@@ -2851,6 +2870,8 @@ def compute_coadd2d(ref_trace_stack, sciimg_stack, sciivar_stack, skymodel_stack
     """
     nimgs, nspec, nspat = sciimg_stack.shape
 
+    # TODO -- If weights is a numpy.ndarray, how can this not crash?
```

Comment:
This tecnically works but raises a warning. Probably we should make the uniform weighting option weights=None.